ci-operator/step-registry/baremetalds/assisted/devscripts/remotenodes/baremetalds-assisted-devscripts-remotenodes-commands.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted devscripts remotenodes command ************"
+## Temp remote_nodes.sh before the dev-scripts PR merges
```

Comment:
As the dev-scripts [PR](https://github.com/openshift-metal3/dev-scripts/pull/1293) is merged, this can be removed.